### PR TITLE
PixelPing: Add to Vulcan pages

### PR DESCRIPTION
**Pull request by jarstelfox**
QA
---

* Load a vulcan page
* Check for a network request to `https://ping.ubreakit.com/?key=ifixit%wiki%2F<SOMEID>%2Fen"`
* Ensure pixel pings for product pages [still match](https://github.com/iFixit/react-commerce/pull/1061#issuecomment-1329540342) `https://ping.ubreakit.com/?key=ifixit%product%2F<SOMEID>%2Fen"`

Closes: https://github.com/iFixit/react-commerce/issues/1790

### CODE REVIEW

**Comment by jarstelfox** [comment]
deploy_block 🌵 on https://github.com/iFixit/ifixit/pull/48590

File: frontend/templates/troubleshooting/index.tsx [hunk 1]
```diff
@@ -185,6 +187,7 @@ const Wiki: NextPageWithLayout<{
             {wikiData.linkedProblems.length > 0 && (
                <RelatedProblems problems={wikiData.linkedProblems} />
             )}
+            <PixelPing id={String(id)} type="troubleshooting" />
```
Comment (jarstelfox) [inline frontend/templates/troubleshooting/index.tsx:190]:
```suggestion
            <PixelPing id={String(id)} type="wiki" />
```

>Shawn Put-It-In-A-Channel Tice :house_with_garden:
[It should be the same as it is currently for the wiki page](https://ifixit.slack.com/archives/CDFNZ4N4A/p1688147216565649?thread_ts=1688081409.739159&cid=CDFNZ4N4A)

**Comment by sctice-ifixit** [comment]
We should use the same identifier that we currently use for the WIKI namespace problem page (i.e., from the perspective of pixel ping stats, there will be no difference between a hit to a /Troubleshooting page and the corresponding WIKI namespace problem page. They're both views on the same data.

File: frontend/components/analytics/PixelPing.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { PIXEL_PING_URL } from '@config/env';
+
+export const PixelPing = ({
+   type,
+   id,
+   site = 'ifixit',
+   lang = 'en',
+}: {
+   type: string;
+   id: string;
```
Comment (djmetzle) [inline frontend/components/analytics/PixelPing.tsx:10]: I think `id` needs to be an `int` or the post will drop that hit.
https://github.com/iFixit/ifixit/blob/178253f6bb078761311a337d13a92bcd4d9dd584/Objects/PixelPingReader.php#L19

Comment (jarstelfox) [reply frontend/components/analytics/PixelPing.tsx:10]: Ah good catch. I'll update the types to help enforce this requirement.

**Comment by deltuh-vee** [comment]
QA 🎬 
Pixel pings are going out for Vulcan pages and are the same as on the corresponding Wiki.

**Review by andyg0808 — APPROVED** [review]
CR :+1: 
This looks reasonable to me! I take it that the troubleshooting API is in fact including the `id` value now? (and that whatever PR might have added that is merged?)

**Comment by jarstelfox** [comment]
> I take it that the troubleshooting API is in fact including the `id` value now? (and that whatever PR might have added that is merged?)

Yep! https://github.com/iFixit/ifixit/pull/48590

**Pull request merged**